ui/assets/css/_header.scss

```diff
@@ -0,0 +1,19 @@
+.usa-disclaimer {
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+  font-size: 12px;
+  padding: 0.25rem 1rem;
+  line-height: 15px;
+}
+
+.navbar {
+  background-color: $color-primary-darkest;
+  height: 80px;
```

Comment:
Yeah possibly, let's see what @micahtaylor thinks. I haven't put any mobile thought into this yet since it still looks the same at narrower resolutions but if we ever have to add anything to the right side of the nav it would change.